tests/aura_processor/test_swap_weth_badger_aura.py

```diff
@@ -74,7 +74,19 @@ def test_swap_weth_for_aura(setup_aura_processor, weth, aura, manager, settlemen
 
   assert settlement.preSignature(uid) > 0
 
-def test_swap_weth_for_aura_must_be_weth_aura(setup_aura_processor, weth, badger, usdc, aura, manager, settlement):
+def test_swap_weth_for_graviaura(setup_aura_processor, weth, bve_aura, manager, settlement):
+  sell_amount = 10000000000000000000 # 10 wETH since there is lower liquidity for graviAURA (fails with 100 wETH)
+
+  order_details = get_cowswap_order(setup_aura_processor, weth, bve_aura, sell_amount)
+
+  data = order_details.order_data
+  uid = order_details.order_uid
+
+  setup_aura_processor.swapWethForAURA(data, uid, {"from": manager})
+
+  assert settlement.preSignature(uid) > 0
```

Comment:
got test passing except one due to a `revert: !cowLowerPrice` in *test_sell_bribes_for_weth_works_when_selling_usdc_for_weth*, which pricer managed to have better, something minor.

This assert made me think on why not making it more strict explicitly, even tho outcome is likely same.Something such as:

```
assert settlement.preSignature(uid) == int( web3.solidityKeccak(["string"],["GPv2Signing.Scheme.PreSign"]).hex(), 16)
```
[ref-1](https://etherscan.io/address/0x9008d19f58aabd9ed0d60971565aa8510560ab41#code#F14#L40) & [ref-2](https://etherscan.io/address/0x9008d19f58aabd9ed0d60971565aa8510560ab41#code#F14#L85)
cc: @sajanrajdev

Reply:
Did you run separately?
Brownie will use the strict pricer even if we set it up with the other due to some obscure bug.

Reply:
<img width="1248" alt="Screenshot 2022-08-10 at 15 53 37" src="https://user-images.githubusercontent.com/13383782/183918978-eece18a1-6589-4b25-afbc-dadff7f7c770.png">